README.md

```diff
@@ -99,7 +99,7 @@ scrape_configs:
         - switch.local # SNMP device.
     metrics_path: /snmp
     params:
-      module: [if_mib]
+      module: [if_mib]  # can't be used as multiple module option. Onle ne module per job allowed
```

Comment:
Spelling: "Can't be used as multiple module option. Only ne module per job allowed"